Report specified amount of slowest tests (0 by default) (#144)
buildkite-plugins/junit-annotate-buildkite-plugin: README.md

The buildkite annotation context to use. Useful to differentiate multiple runs of this plugin in a single pipeline.

### `report-slowest` (optional)
Default: `0`

> Adds a table with the specified amount of slowest tests listed in the JUnit XML files. The annotation will also be shown when all tests are successful.

Reviewer: What do you think of:
```suggestion
Include the specified number of slowest tests in the annotation. The annotation will always be shown.
```

Author: Sure looks good!
